installer/src/panels/com/jboss/devstudio/core/installer/bean/P2IU.java

```diff
@@ -14,16 +14,18 @@
 	private String description;
 	private Boolean selected;
 	private String path;
+	private String size;  // estimated size - needed only in summary panel
 	
-	public P2IU(Boolean selected, String id, String label, String description, String path) {
+	public P2IU(Boolean selected, String id, String label, String description, String path, String size) {
```

Comment:
This should break tests code here https://github.com/jbdevstudio/jbdevstudio-product/tree/master/installer/src/test/com/jboss/devstudio/core/installer. This is just my guess, I did not really try it in eclipse. Tests probably are not not compiled during the build and not executed :)